code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -100,19 +94,25 @@
 	if (mode.check_lowpop_lowimpact_injection())
 		return FALSE
 
+	if (!..()) // calls check_candidates()
+		return FALSE
+
 	return TRUE
 
-/datum/dynamic_ruleset/midround/from_ghosts/execute()
+/datum/dynamic_ruleset/midround/from_ghosts/execute(forced = FALSE)
 	var/list/possible_candidates = list()
 	possible_candidates.Add(dead_players)
 	possible_candidates.Add(list_observers)
-	send_applications(possible_candidates)
-	return length(assigned)
+	send_applications(possible_candidates, forced)
+	return length(assigned) ? TRUE : NOT_ENOUGH_PLAYERS
```

Comment:
I am unsure if mixing types is a good idea here.
Maybe add success define and return it for the sake of clarity?

Reply:
They're both numbers